apps/web/src/server/app/validation/__dev__/local-planning-departments.js

```diff
@@ -0,0 +1,2361 @@
+
+export default [
```

Comment:
This folder and file will be short lived until the API works right?

Reply:
Yes